identity/src/commonMain/kotlin/com/android/identity/document/Document.kt

```diff
@@ -258,7 +258,6 @@ class Document private constructor(
             // that secure area implementations do not use IllegalArgumentException for transient
             // errors (like server connections).
             Logger.e(TAG, "Error accessing $credentialType ${credential.identifier}, deleting it", err)
-            credential.delete()
```

Comment:
@sorotokin, plz review this. You added it recently.

Reply:
OK, let's deal with that bug separate. Really like to land the Base64 fix.